pytest_splunk_addon/plugin.py

```diff
@@ -116,6 +120,63 @@ def pytest_sessionstart(session):
         sample_generator.get_samples(store_events)
 
 
+def pytest_sessionfinish(session, exitstatus):
+    try:
+        with open(".{0}splunk_type.txt".format(os.sep), "r") as splunk_type_file:
+            splunk_data = [line.rstrip() for line in splunk_type_file]
```

Comment:
Is it possible to unpack splunk_data and assign more readable names to it? Later we are using `splunk_data[0]`, `splunk_data[1]`, `splunk_data[2]`, but it's not clear what is intended to be there.

Reply:
Yes, unpacked splunk_data and assigned more readable names to get more clear insinght.